pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -210,6 +210,9 @@ type KudoOperatorTaskSpec struct {
 	// a specific operator version in the official repo, defaults to the most recent one
 	// +optional
 	OperatorVersion string `json:"operatorVersion,omitempty"`
+	// a parameter file name that will be used to populate Instance.Spec.Parameters
```

Comment:
```suggestion
	// name of the template (i.e. key in `OperatorVersionSpec.Templates`) from the *parent* instance used to generate a parameter file used to populate the *child* Instance.Spec.Parameters
```

Let's be precise here, since this is somewhat subtle.

Reply:
I rephrased it a little but the gist is there